packages/teams-js/test/public/media.spec.ts

```diff
@@ -838,7 +713,146 @@ describe('media', () => {
           },
         } as DOMMessageEvent);
 
-        return expect(promise).rejects.toEqual({ errorCode: ErrorCode.SIZE_EXCEEDED });
+        await expect(promise).rejects.toMatchObject({ errorCode: ErrorCode.SIZE_EXCEEDED });
```

Comment:
I think the new standard is to use .toEqual for comparing SdkError objects.

Reply:
you're right!